### PR TITLE
Update getFullName function to be more consistent

### PR DESCRIPTION
**Pull request by olikami**
Display the name in the header of hc more consistent and in-line with MiData.

### CODE REVIEW

**Comment by tobijuon** [comment]
Hi @olikami 
Thanks for your pull request, seems like a small beautification.
@TaminoWalter can you check it and merge it into the main branch?

File: src/app/shared/models/person.ts [hunk 1]
```diff
@@ -21,6 +21,9 @@ export class Person {
   }
 
   public getFullName(): string {
+    if (this.nickname) {
```
Comment (TaminoWalter) [inline src/app/shared/models/person.ts:24]: @olikami this seems to have a typo in it. It should be `this.nickName`

**Comment by TaminoWalter** [comment]
@olikami I merged your changes to a working branch of myself and I will fix the typo for you

**Comment by olikami** [comment]
Perfect, thanks

**Pull request merged**